javademo/src/main/java/com/example/javademo/Main.java

```diff
@@ -24,7 +24,7 @@ public static void main(String[] args) throws Exception {
         // ⚠️ WARNING ⚠️ The key generated here is just for demonstration purposes.
         // DO NOT GENERATE YOUR KEYS THIS WAY. USE CRYPTOGRAPHICALLY SAFE TOOLS!
         Felt privateKey = new Felt(ThreadLocalRandom.current().nextLong(1, Long.MAX_VALUE));
-        Provider provider = GatewayProvider.makeTestnetClient();
+        Provider provider = GatewayProvider.makeTestnetProvider();
```

Comment:
Problem is, this way Javademo is up to date with changes, but cannot be run if using package version specified in its build.gradle.